MIGRATION.md

```diff
@@ -61,6 +61,32 @@ As `o-colors` [no longer outputs usecase CSS classes](#MIGRATION.md#migrating-fr
 ));
 ```
 
+#### oColorsGetTextColor
+
+The name of the first argument of `oColorsGetTextColor` has changed. It was `$backgroundd` (with a double `d`) but it now spelled correctly as `$background`. E.g:
+
+```diff
+-	color: oColorsGetTextColor($backgroundd: 'paper');
++	color: oColorsGetTextColor($background: 'paper');
+```
+
+`oColorsGetTextColor` now errors if the contrast between the given background and text colour does not pass WCAG 2.1 level AA for [normal text](https://www.w3.org/TR/WCAG21/#contrast-minimum). Previously it would only throw a warning provided the contrast check passed for [large text](https://www.w3.org/TR/WCAG21/#dfn-large-scale). Accordingly, the third `$warnings` argument is now `$minimum-contrast`. It accepts a number, the minimum contrast allowed between the given background and the returned text colour. To migrate update the third `$warnings` argument of `oColorsGetTextColor` to `$minimum-contrast`:
+- **not set**: no changes are needed unless contrast errors are thrown. If an error is thrown set to `3`, to error if the colour contrast is not enough for [large text](https://www.w3.org/TR/WCAG21/#dfn-large-scale), or `null` to ignore the contrast of the resulting text colour.
```

Comment:
"if an error is thrown set to `3`, to error" is confusing

Reply:
_yes_,thanks I'll give the wording of that another go!